tests/test_pythoncapi_compat.py

```diff
@@ -62,7 +62,7 @@ def build_ext():
     display_title("Build test extensions")
     if os.path.exists("build"):
         shutil.rmtree("build")
-    cmd = [sys.executable, "setup.py", "build"]
+    cmd = [sys.executable, "setup.py", "build", "--debug"]
```

Comment:
Please create a separated PR to add ``--debug``. It might explain the build failure on Windows:

```suggestion
    cmd = [sys.executable, "setup.py", "build"]
```